src/test/java/steps/WebSteps.java

```diff
@@ -0,0 +1,41 @@
+package steps;
+
+import factoryPattern.PageFactory;
+import factoryPattern.PageTypeEnums;
+import factoryPattern.TestedPage;
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.thucydides.core.annotations.Step;
+import org.openqa.selenium.Keys;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class WebSteps extends AbstractSteps {
+
+    @Step
+    public void openPageByParameter(final String nameOfPage) {
```

Comment:
you may use PageTypeEnums here instead of String type

Reply:
fixed